### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by NotActuallyErik** [Belieal/flipping-utilities#45]
First of all, great job on the plugin! I've been trying it out today and I'm really enjoying it so far! 

Here are a few suggestions for possible improvements:

- A feature that calculates the profit/each for any arbitrary item over x-amount of transactions given varying buy/sell prices would be cool! Alternatively, this could be adjusted manually by the user

- Status-bars such as the one on Runelites own GE, the framework already exists so it should be sorta simple-ish to add. Maybe using the GrandExchangeOffersPanel-class is possible?

- A session and/or total profit counter would be useful

- The panels containing the item information in the toolbar feel very large and somewhat cumbersome compared to a lot of other RL plugins. An option to customize the look of the panels would be neat

I'll look into these myself as soon as I've familiarized myself with the code! :)

**Comment by Belieal** [comment]
Thank you for these suggestions and I'm happy you're enjoying the plugin! :)

1. I'm not entirely certain what you mean here, but as far as I understand the statistics tab will address this. (see #35 and #41)
2. I really like this suggestion! I think it would be super cool to see your offer progress on the flipping panel. I don't think we can access the GEOffersPanel from our external plugin, but we could definitely emulate what it does with our current framework.
3. This will also be addressed in the statistics tab. :)
4. I get what you mean here. I've recently also found it pretty annoying how large the panels are, so I've started transitioning most of the font types into small fonts along with downsizing item icons (16x16 px vs 32x32 px). These will come with user options to configure back to the regular fontstyle.

Again, thank you so much for these suggestions and I really look forward to and appreciate any contributions. I'll always be happy to explain any of the code, if need be, as well. :)
